app/views/items/index.html.erb

```diff
@@ -10,14 +10,20 @@
       <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
       <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
       <%= link_to image_tag("#{item.image}"), "/items/#{item.id}" %>
-      <p> <%= item.description unless @merchant%> </p>
```

Comment:
This `unless` was given to us in the source code but it contradicted the user story so I removed it. This caused some of the given tests to break which I fixed.

Reply:
Nice catch....wish we had more time to ask the instructors.